Code/Editor/TrackView/TrackViewNodes.cpp

```diff
@@ -2370,7 +2372,7 @@ bool CTrackViewNodesCtrl::FillAddTrackMenu(STrackMenuTreeNode& menuAddTrack, con
 
         // only add tracks to the that STrackMenuTreeNode tree that haven't already been added
         CTrackViewTrackBundle matchedTracks = animNode->GetTracksByParam(paramType);
-        if (matchedTracks.GetCount() == 0)
+        if (matchedTracks.GetCount() == 0 && !splittedName.isEmpty())
```

Comment:
question: Shouldn't we never have to check `splittedName.isEmpty()` as otherwise we'd never enter the loop? (`for (const QString& segment : splittedName)` above)